components/KiboAddressForm.vue

```diff
@@ -6,65 +6,91 @@
         name="firstName"
         class="form__element form__element--half"
         required
+        :valid="!errors.firstName"
+        :error-message="errors.firstName"
         :value="addressDetails.firstName"
         @input="updateField('firstName', $event)"
+        @blur="validate('firstName')"
       />
       <SfInput
         :label="$t('Last Name')"
         name="lastName"
         class="form__element form__element--half form__element--half-even"
         required
+        :valid="!errors.lastNameOrSurname"
+        :error-message="errors.lastNameOrSurname"
         :value="addressDetails.lastNameOrSurname"
         @input="updateField('lastNameOrSurname', $event)"
+        @blur="validate('lastNameOrSurname')"
       />
       <SfInput
         :label="$t('Street Address')"
         name="streetName"
         class="form__element"
         required
+        :valid="!errors.address1"
+        :error-message="errors.address1"
         :value="addressDetails.address && addressDetails.address.address1"
         @input="updateField('address1', $event)"
+        @blur="validate('address1')"
       />
       <SfInput
         :label="$t('Apt')"
         name="apartment"
         class="form__element"
         required
+        :valid="!errors.address2"
+        :error-message="errors.address2"
         :value="addressDetails.address && addressDetails.address.address2"
         @input="updateField('address2', $event)"
+        @blur="validate('address2')"
       />
       <SfInput
         :label="$t('City')"
         name="city"
         class="form__element form__element--half"
         required
+        :valid="!errors.cityOrTown"
+        :error-message="errors.cityOrTown"
         :value="addressDetails.address && addressDetails.address.cityOrTown"
         @input="updateField('cityOrTown', $event)"
+        @blur="validate('cityOrTown')"
       />
       <SfInput
         :label="$t('State')"
         name="state"
         class="form__element form__element--half form__element--half-even"
         required
+        :valid="!errors.stateOrProvince"
+        :error-message="errors.stateOrProvince"
         :value="addressDetails.address && addressDetails.address.stateOrProvince"
         @input="updateField('stateOrProvince', $event)"
+        @blur="validate('stateOrProvince')"
       />
       <SfInput
         :label="$t('Zip Code')"
         name="zipCode"
         class="form__element form__element--half"
         required
+        :valid="!errors.postalOrZipCode"
+        :error-message="errors.postalOrZipCode"
         :value="addressDetails.address && addressDetails.address.postalOrZipCode"
         @input="updateField('postalOrZipCode', $event)"
+        @blur="validate('postalOrZipCode')"
       />
       <SfSelect
         :label="$t('Country')"
         :placeholder="`Select ${$t('Country')}`"
```

Comment:
"Select country" both words should be localized ?

Reply:
Resolved.